config/locales/en.yml

```diff
@@ -55,7 +55,6 @@ en:
       conversation_creation: "[New conversation] - #%{display_id} has been created in %{inbox_name}"
       conversation_assignment: "[Assigned to you] - #%{display_id} has been assigned to you"
       assigned_conversation_new_message: "[New message] - #%{display_id} %{content}"
-      conversation_mention: "You have been mentioned in conversation [ID - %{display_id}] by %{name}"
```

Comment:
Isn't this text used in the email notifications?

Reply:
No. I have verified by searching the entire project using the code ```notifications.notification_title. conversation_mention```

Reply:
I looked at the `conversation_reply_mailer`, it is not translated. 

I would suggest keeping this text for now instead of deleting it as the translation would also get deleted. It is an unnecessary effort from the  I will send a follow-up PR moving this text.